python/pyspark/errors/__init__.py

```diff
@@ -45,4 +50,11 @@
     "SparkUpgradeException",
     "PySparkTypeError",
     "PySparkValueError",
+    "SparkConnectException",
+    "SparkConnectGrpcException",
+    "SparkConnectAnalysisException",
+    "SparkConnectParseException",
+    "SparkConnectTempTableAlreadyExistsException",
+    "PySparkTypeError",
+    "PySparkValueError",
```

Comment:
```suggestion
```

`PySparkTypeError` and `PySparkValueError` are duplicated here

Reply:
Oops. Thanks for the correction!